### ISSUE TITLE
getting wrong date for "10-Apr-15"

### ISSUE THREAD

**Issue by Saini-96** [facebook/duckling#444]
```json
{
  "_text": "10-Apr-15",
  "entities": {
    "datetime": [
      {
        "confidence": 1,
        "values": [
          {
            "value": "1985-04-01T00:00:00.000-08:00",
            "grain": "month",
            "type": "value"
          }
        ],
        "value": "1985-04-01T00:00:00.000-08:00",
        "grain": "month",
        "type": "value"
      }
    ]
  },
  "msg_id": "11MWpfpIyUjICxRv0"
}
```

**Comment by chessai** [comment]
From debug output:

```
haxlsh> H.io $ debug (makeLocale EN Nothing) "10-Apr-15" [Seal Time]
intersect (Apr-15)
-- April (Apr)
-- -- regex (Apr)
-- year (latent) (-15)
-- -- negative numbers (-15)
-- -- -- regex (-)
-- -- -- integer (numeric) (15)
-- -- -- -- regex (15)
[Entity {dim = "time", body = "Apr-15", value = RVal Time (TimeValue (SimpleValue (InstantValue {vValue = 1985-04-01 00:00:00 -0200, vGrain = Month})) [SimpleValue (InstantValue {vValue = 1985-04-01 00:00:00 -0200, vGrain = Month})] Nothing), start = 3, end = 9, latent = False, enode = Node {nodeRange = Range 3 9, token = Token Time TimeData{latent=False, grain=Month, form=Nothing, direction=Nothing, holiday=Nothing, hasTimezone=False}, children = [Node {nodeRange = Range 3 6, token = Token Time TimeData{latent=False, grain=Month, form=Just (Month {month = 4}), direction=Nothing, holiday=Nothing, hasTimezone=False}, children = [Node {nodeRange = Range 3 6, token = Token RegexMatch (GroupMatch []), children = [], rule = Nothing}], rule = Just "April"},Node {nodeRange = Range 6 9, token = Token Time TimeData{latent=True, grain=Year, form=Nothing, direction=Nothing, holiday=Nothing, hasTimezone=False}, children = [Node {nodeRange = Range 6 9, token = Token Numeral (NumeralData {value = -15.0, grain = Nothing, multipliable = False, okForAnyTime = True}), children = [Node {nodeRange = Range 6 7, token = Token RegexMatch (GroupMatch ["-"]), children = [], rule = Nothing},Node {nodeRange = Range 7 9, token = Token Numeral (NumeralData {value = 15.0, grain = Nothing, multipliable = False, okForAnyTime = True}), children = [Node {nodeRange = Range 7 9, token = Token RegexMatch (GroupMatch ["15"]), children = [], rule = Nothing}], rule = Just "integer (numeric)"}], rule = Just "negative numbers"}], rule = Just "year (latent)"}], rule = Just "intersect"}}]
```

Confirms your issue. Not looking into this right now, just providing more context.

**Comment by rabinpoudyal** [comment]
Same issue here
```
curl -XPOST http://0.0.0.0:8000/parse --data 'locale=en_US&text=Nov 18-20'                                                                            ─╯
[{"body":"Nov 18-20","start":0,"value":{"values":[{"to":{"value":"2020-11-21T00:00:00.000-08:00","grain":"day"},"from":{"value":"2020-11-18T00:00:00.000-08:00","grain":"day"},"type":"interval"},{"to":{"value":"2021-11-21T00:00:00.000-08:00","grain":"day"},"from":{"value":"2021-11-18T00:00:00.000-08:00","grain":"day"},"type":"interval"},{"to":{"value":"2022-11-21T00:00:00.000-08:00","grain":"day"},"from":{"value":"2022-11-18T00:00:00.000-08:00","grain":"day"},"type":"interval"}],"to":{"value":"2020-11-21T00:00:00.000-08:00","grain":"day"},"from":{"value":"2020-11-18T00:00:00.000-08:00","grain":"day"},"type":"interval"},"end":9,"dim":"time","latent":false}]%
```

**Comment by chessai** [comment]
Closed by 78904b66804e435d4fe0fc595b6d45c19553651a
